modules/liveIntentIdSystem.js

```diff
@@ -217,6 +217,10 @@ export const liveIntentIdSubmodule = {
         result.sovrn = { 'id': value.sovrn }
       }
 
+      if (value.thetradedesk) {
+        result.thetradedesk = { 'id': value.thetradedesk }
+      }
+
```

Comment:
I submitted a PR to also expose the ext.provider in v7 now: https://github.com/prebid/Prebid.js/pull/10843/files#diff-489289275f4ee5264024b54c9c851e6c57684fd2d0bccab91d8b9ccef150e235R198. We should do it here too.